pkg/scripts/environment_test.go

```diff
@@ -40,3 +42,38 @@ func equalArrayContents(a []string, b []string) bool {
 	}
 	return true
 }
+
+func TestConvertEnvironmentToDocker(t *testing.T) {
+	testValues := []string{
+		"Value1",
+		"$Value1",
+		"${Value1}",
+		"`Value1`",
+		`"`,
+	}
+	var charValues []string
+	for ch := rune(32); ch < 127; ch++ {
+		charValues = append(charValues, "AB"+string(ch)+"CD")
+	}
+	for _, testValue := range append(testValues, charValues...) {
+		list := api.EnvironmentList{{Name: "TEST", Value: testValue}}
+		converted := ConvertEnvironmentToDocker(list)
+		builder := imagebuilder.NewBuilder(nil)
+		parsed, err := imagebuilder.ParseDockerfile(bytes.NewBufferString(converted + "\n"))
```

Comment:
@nalind I'm reluctant to add a dependency to `imagebuilder` when we also have a dep on docker/docker - especially for just a test.
Can you refactor to use https://github.com/openshift/moby-moby/blob/openshift-3.10-docker-b68221c/builder/dockerfile/parser/parser.go#L259 ?